jiant/proj/simple/runscript.py

```diff
@@ -102,11 +103,18 @@ def run_simple(args: RunConfiguration):
     with distributed.only_first_process(local_rank=args.local_rank):
         # === Step 1: Write task configs based on templates === #
         full_task_name_list = sorted(list(set(args.train_tasks + args.val_tasks + args.test_tasks)))
-        task_config_path_dict = create_and_write_task_configs(
-            task_name_list=full_task_name_list,
-            data_dir=args.data_dir,
-            task_config_base_path=os.path.join(args.exp_dir, "task_configs"),
-        )
+        task_config_path_dict = {}
+        if args.create_config:
+            task_config_path_dict = create_and_write_task_configs(
```

Comment:
I think we should deprecate `create_and_write_task_configs` and just use downloader configs from here on.

Reply:
Out of scope for these changes but I agree. Opened an issue here: https://github.com/jiant-dev/jiant/issues/170